systemtest/src/test/resources/io/strimzi/systemtest/KafkaST.testSendMessagesTlsAuthenticated.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: kafka.strimzi.io/v1alpha1
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    replicas: 3
+    listeners:
+      tls: {}
```

Comment:
This is called `KafkaST.testSendMessagesTlsAuthenticated.yaml` but `authentication` is not enabled here. That is a bit confusing.

Reply:
Good point. Let me try to add authentication then.